picoquic/picoquic_internal.h

```diff
@@ -683,7 +684,7 @@ int picoquic_prepare_misc_frame(picoquic_misc_frame_header_t* misc_frame, uint8_
 
 /* send/receive */
 
-int picoquic_decode_frames(picoquic_cnx_t* cnx, uint8_t* bytes,
+int picoquic_decode_frames(picoquic_cnx_t* cnx, const uint8_t* bytes,
     size_t bytes_max, int restricted, uint64_t current_time);
 
```

Comment:
See remark above. I would leave that signature unchanged in the first PR, to contain the size and scope of that PR.

Reply:
Right.  I just hated to write new functions lacking const and go back and redo them later.  But you are correct -- too many unrelated themes in a single PR.